tools/sof-dump-status.py

```diff
@@ -366,8 +402,15 @@ def dump_dapm(dapm, filter = "all"):
     sysinfo = clsSYSCardInfo()
     if ret_args['platform'] is True:
         sysinfo.loadPCI()
+        mach_name = None
         for pci_info in sysinfo.pci_lst:
-            print(pci_info['hw_name'])
+            mach_name = pci_info['hw_name']
```

Comment:
It looks like `mach-name` will simply be equal to the last element `sysinfo.pci_lst[-1]['hw_name']`. So what does this `for` loop achieve?

Is there an indentation issue maybe?

Reply:
I have no idea, this is what I started from. This script needs a lot of work, but for now the problem is to solve conceptual problems coming from PCI and ACPI confusions. How the code is written is a separate topic entirely.